app/models/manageiq/providers/ansible_tower/inventory/persister/automation_manager.rb

```diff
@@ -1,19 +1,18 @@
 class ManageIQ::Providers::AnsibleTower::Inventory::Persister::AutomationManager < ManagerRefresh::Inventory::Persister
+  include ManagerRefresh::Inventory::AutomationManager
+
+  has_inventory automation_manager.credentials
```

Comment:
yes, the block/proc/lambda does delay it

but it is the part outside the lambda that concerns me.